docs/02-get-started/01-quick-install.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Quick install
+---
+
+To get started with Kyma, let's quickly install it first.
+
+## Install Kyma
+
+To install Kyma on a local k3d cluster, run:
+
+```bash
+kyma provision k3d
+kyma deploy
+```
+
```

Comment:
I need to choose whether I want the Kyma certificate to be installed locally. Should recommend yes or no in the in the guide?

Reply:
I assumed this was not the object of the guide and was up to the user, but it's a good question. I also assumed the user would probably want it and chose yes every time. I'll check.

Reply:
Yes, the user will need it. I'll add a note.

Reply:
Added a line, as there's already a note below and I didn't want to cramp it. 
![image](https://user-images.githubusercontent.com/53601578/133637666-c481c214-727e-4200-9986-66960a62bed5.png)